### PR TITLE
search

### PR DESCRIPTION
**Pull request by Kateryna199420**
- [DEMO LINK](https://<Kateryna199420>.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://<Kateryna199420>.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -25,8 +25,8 @@ ___
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_search-bar-airbnb/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_search-bar-airbnb/report/html_report/)
+- [DEMO LINK](https://<Kateryna199420>.github.io/layout_search-bar-airbnb/)
+- [TEST REPORT LINK](https://<Kateryna199420>.github.io/layout_search-bar-airbnb/report/html_report/)
```
Comment (volodymyr-soltys97) [inline readme.md:29]: Need to fix the links and update it in the PR description
```suggestion
- [DEMO LINK](https://Kateryna199420.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://Kateryna199420.github.io/layout_search-bar-airbnb/report/html_report/)
```

File: src/index.html [hunk 2]
```diff
@@ -11,6 +11,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+    class="search__bar big__bar"
+    method="post"
+    data-qa="big"
+  >
```
Comment (volodymyr-soltys97) [inline src/index.html:19]: Need to fix code style everywhere
```suggestion
    <form
      class="search__bar big__bar"
      method="post"
      data-qa="big"
    >
```

File: src/index.html [hunk 3]
```diff
@@ -11,6 +11,44 @@
...
+      <label for="biginput" class="big">
+        <img src="./images/Search.svg" alt="search"
+        class="big__icon">
```
Comment (volodymyr-soltys97) [inline src/index.html:22]: img is redundant here, try to add image as a background
```suggestion
```

File: src/index.html [hunk 4]
```diff
@@ -11,6 +11,44 @@
...
+        <input class="search__input big--input" type="text"
+        name="biginput"
+        data-qa="keypress"
+        placeholder="Try &quot;Los Angeles&quot;"
```
Comment (volodymyr-soltys97) [inline src/index.html:26]:
```suggestion
        <input
          class="search__input big--input"
          type="text"
          name="biginput"
          data-qa="keypress"
          placeholder="Try &quot;Los Angeles&quot;"
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your code

File: readme.md (hunk 1 above)
Comment (sTorba24) [reply readme.md:29]: image
Still not fixed, also update you PR description.

File: src/index.html [hunk 5]
```diff
@@ -11,6 +11,37 @@
...
+      method="post"
+      data-qa="big"
+    >
+      <label for="biginput" class="big">
+
+        <input class="search__input big--input" type="text"
+         name="biginput"
+         data-qa="keypress"
+         placeholder="Try &quot;Los Angeles&quot;"
+        >
```
Comment (sTorba24) [inline src/index.html:26]:
```suggestion
        <input
          class="search__input big--input"
          type="text"
          name="biginput"
          data-qa="keypress"
          placeholder="Try &quot;Los Angeles&quot;"
        >
```

Comment (sTorba24) [reply src/index.html:26]: every attribute should be on new line if you started moving it

File: src/style.css [hunk 6]
```diff
@@ -1 +1,95 @@
...
+  --text-color: #3d4e61;
+  --color: #687c94;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+form {
```
Comment (sTorba24) [inline src/style.css:22]:
```suggestion
.search__bar {
```

File: src/style.css [hunk 7]
```diff
@@ -1 +1,95 @@
...
+  margin: 20px 0;
+}
+
+.big--input {
+  padding-left: 62px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background-color: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  font-family: Avenir, sans-serif;
```
Comment (sTorba24) [inline src/style.css:32]: set it for body or html and use inherit value here

File: src/style.css [hunk 8]
```diff
@@ -1 +1,95 @@
...
+  background-image: url(images/Search.svg);
+  background-position:15px;
+  background-repeat: no-repeat;
+  background-size: 11px;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  width: 100%;
+  height: 42px;
+  flex-shrink: 0;
+
+}
```
Comment (sTorba24) [inline src/style.css:70]:
```suggestion
  flex-shrink: 0;
}
```

File: src/style.css [hunk 9]
```diff
@@ -1 +1,95 @@
...
+
+
+
+.search__input::placeholder {
+  font-family: Avenir, sans-serif;
+  color: var(--text-color);
+  font-weight: 300;
+
```
Comment (sTorba24) [inline src/style.css:78]:
```suggestion
```

**Review by sTorba24 — CHANGES_REQUESTED** [review]
Almost good! Lets improve it, fix comments from previous review

**Pull request closed without merging**